backend/src/entities/member.entity.ts

```diff
@@ -0,0 +1,201 @@
+import {
+    Entity,
+    Column,
+    ManyToOne,
+    OneToMany,
+    ManyToMany,
+    JoinColumn,
+    JoinTable,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+  } from 'typeorm';
+import { MemberType } from './memberType.entity';
+import { Ministry } from './ministry.entity';
+import { ContactLog } from './contactLog.entity';
+import { MemberUnderMinister } from './memberUnderMinister.entity';
+  
+  export enum MaritalStatus {
+    Married = 'Married',
+    Single = 'Single',
+    Divorced = 'Divorced',
+    Widowed = 'Widowed',
+  }
+
+  export enum Gender {
+    Male = 'Male',
+    Female = 'Female',
+  }
+
+  @Entity()
+  export class Member {
+    @PrimaryGeneratedColumn()
+    id: number;
+  
+    @ManyToOne(() => MemberType, (memberType) => memberType.members)
+    @JoinColumn({ name: 'member_type_id' })
+    memberType: MemberType;
+
+    @ManyToMany(() => Ministry, ministry => ministry.members)
+    @JoinTable({
+      name: 'member_ministries',
+      joinColumn: { name: "member_id" },
+      inverseJoinColumn: { name: "ministry_id" }
+    })
+    ministries: Ministry[];
```

Comment:
each member-ministry entry should also have a role associated with it.

member_ministries table will have
- member_id
- ministry_id
- role_id